caas/kubernetes/provider/k8s.go

```diff
@@ -65,6 +65,8 @@ import (
 var logger = loggo.GetLogger("juju.kubernetes.provider")
 
 const (
+	DefaultNamespace = "default"
```

Comment:
Needs comment if exported, but no need to export
Actually, I'd still like a comment to explain what this is used for because unless you know the semantics of the code that uses it (which is in another file), it's not apparent. Maybe even move this to near where it's used.

Reply:
Done & Done